admin/controllers/images/views/show/model.php

```diff
@@ -6,7 +6,18 @@
     array_push($valid_image_types, "'$type'");
 }
 
-$all_images = DB::fetchall("select * from media where mimetype in (" . implode(",", $valid_image_types) . ") ORDER BY id DESC");
+
+$searchtext = Input::getvar('searchtext','TEXT',null);
```

Comment:
nit pick, third argument `null` is not needed here since it is null by default

Reply:
my own fault, I _did_ ask for a code review. ;)

gonna leave it as is (because lazy) - but explicit is never a bad thing, and can even help catch bugs if the interface changes (when would that ever happen? ;) )